src/tactic/converter/interactive.lean

```diff
@@ -112,6 +112,19 @@ We use this tactic for writing tests.
 meta def guard_target (p : parse texpr) : conv unit :=
 do `(%%t = _) ← target, tactic.interactive.guard_expr_eq t p
 
+/--
+When using `rw` in `conv` mode on conditional equality lemmas, sometimes `⊢` goals can be
+introduced which are impossible to close with `conv` tactics. Note that goals like `⊢ a = 0` will
+be displayed by conv mode as `| a`, making the problem difficult to diagnose at times.
```

Comment:
Is this fixable, by checking whether the RHS is a metavariable when displaying the goal?

Reply:
I think the fix would have to be within lean itself. That would certainly be better than what happens today.